### ISSUE TITLE
Postman stuck on an infinite loading screen

### ISSUE THREAD

**Issue by AntogamerYT** [a85/POSTMan-Chrome-Extension#12319]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Postman is stuck in an infinite loading screen and I can only close it via the task manager, Alt F4 and closing it in the taskbar don't work

### Steps To Reproduce

1. Open Postman
2. That's it, watch it go into full screen mode and load for eternity

### Screenshots or Videos

https://github.com/postmanlabs/postman-app-support/assets/64664639/12b11698-055f-4bb6-bf49-aa14710cdc41

![devtools window](https://github.com/postmanlabs/postman-app-support/assets/64664639/0833ab2f-6300-4a29-8c91-24eaead63fc7)

^ this is pretty interesting because it seems to block a js script from running and I think that is the main issue

### Operating System

Windows 11 22H2 (22621.2134)

### Postman Version

i think 10.17.4 (by looking at %localappdata%\Postman)

### Postman Platform

Postman App

### User Account Type

Signed In User

### Additional Context?

I've tried uninstalling and reinstalling the Postman app but it didn't change anything

**Comment by dmitrevnik** [comment]
same

**Comment by AntogamerYT** [comment]
bump

**Comment by jefflinhq** [comment]
I have the same problem.

**Comment by deliro** [comment]
Don't want to be "that guy" but why did you ruin a good tool? (For sure I also have loading issues)

**Comment by amcastror** [comment]
Same problem here..

**Comment by AntogamerYT** [comment]
I've found a smart fix:
- Download [Revo Uninstaller](https://www.revouninstaller.com/products/revo-uninstaller-free/) or any custom uninstaller with a way to do an in-depth cleaning of an app after uninstalling it
- Search for postman, uninstall it (PRO TIP: Disable the system restore point creation if you want to save time, you can disable it in Options->Uninstaller->Make a System Restore Point before uninstall)
- After postman's uninstaller finished, continue with the advanced scan on Revo and delete every registry key & files
- Reinstall postman
- Profit

Hope this helped

**Comment by amcastror** [comment]
It looks like my problem was a DNS problem. After restarting the router postman worked.

**Comment by AntogamerYT** [comment]
> It looks like my problem was a DNS problem. After restarting the router postman worked.

it probably was related to some local files for me, according to the devtools (console, "inspect element"..) of postman. 
I've had this issue for months, i flushed my dns cache lots of times during this period and also restarted the router a lot but i got tired of it to the point that i switched to insomnia; very valid alternative, but i still missed postman

**Comment by amcastror** [comment]
> > It looks like my problem was a DNS problem. After restarting the router postman worked.
> 
> it probably was related to some local files for me, according to the devtools (console, "inspect element"..) of postman. I've had this issue for months, i flushed my dns cache lots of times during this period and also restarted the router a lot but i got tired of it to the point that i switched to insomnia; very valid alternative, but i still missed postman

Nice! I'll take a look at insomnia, looks great. Thanks.

**Comment by AradAryan** [comment]
it can be DNS or network Problem
change your DNS and test it, i accedently changed my DNS and my problem solved :)

**Comment by Suliman-7** [comment]
I faced the same problem, I just sign out then sign in and it's work !

**Comment by SAray2013** [comment]
> I faced the same problem, I just sign out then sign in and it's work !

Thank you. It's work.

**Comment by safakferhatkaya** [comment]
> I faced the same problem, I just sign out then sign in and it's work !

Ah thanks!

**Comment by Utkarsh-Palav** [comment]
> I faced the same problem, I just sign out then sign in and it's work !

It worked. Thank you bro

**Comment by ashimaro** [comment]
Solution that works for me  : End task for Postman at task manager

**Comment by devrahuljourney** [comment]
> I've found a smart fix:
> 
> * Download [Revo Uninstaller](https://www.revouninstaller.com/products/revo-uninstaller-free/) or any custom uninstaller with a way to do an in-depth cleaning of an app after uninstalling it
> * Search for postman, uninstall it (PRO TIP: Disable the system restore point creation if you want to save time, you can disable it in Options->Uninstaller->Make a System Restore Point before uninstall)
> * After postman's uninstaller finished, continue with the advanced scan on Revo and delete every registry key & files
> * Reinstall postman
> * Profit
> 
> Hope this helped

Damn ! this helped me to resolve my issue. i tried by manually uninstalling the postman but it didn't work . Thanks brother .

**Comment by aleksandarbos** [comment]
`Sign Out -> Sign In again` <- that solved it in my case. seems like my session has expired but it had to be refreshed manually.

**Comment by jhabaa** [comment]
> `Sign Out -> Sign In again` <- that solved it in my case. seems like my session has expired but it had to be refreshed manually.

Crazy, but it works. Thanks

**Comment by zeeshan-shabbir** [comment]
i dont want to sign out it will loss my local data. there is some unsave request and collection.
this happen after i update to newer version  Postman v11.13.1


https://github.com/user-attachments/assets/43668b90-d35e-4a32-a27d-461ddbafd5b1

**Comment by arosicki** [comment]
Got similar issue recently (I wasn't able to sign in and out, entire app just froze)

> I've found a smart fix:
>
> Download [Revo Uninstaller](https://www.revouninstaller.com/products/revo-uninstaller-free/) or any custom uninstaller with a way to do an in-depth cleaning of an app after uninstalling it
> Search for postman, uninstall it (PRO TIP: Disable the system restore point creation if you want to save time, you can disable it in Options->Uninstaller->Make a System Restore Point before uninstall)
> After postman's uninstaller finished, continue with the advanced scan on Revo and delete every registry key & files
> Reinstall postman
> Profit
> Hope this helped

There's a faster option
You can go to `%appdata%` (I guess similar folder exists on mac) and delete `Postman` folder

If you don't want to loose your local data you can make a copy before deleting and then try pasting some files there and see what happens (I haven't tried that, but I would start by pasting some storage related files)

You also can go other way around and start deleting files from there to see which one is causing the issue.

If nothing works in worst case you can restore the broken data and wait for the fix 🙃

**Comment by guccikom** [comment]
> I faced the same problem, I just sign out then sign in and it's work !

Yeap. Sign out then back in, fixed my problem, thanks!!

**Comment by briankrupp** [comment]
I ran into this issue recently. Tried to sign out and sign back in, but that didn't work. I'm starting to look at other solutions though as Postman has become less reliable.

**Comment by AntogamerYT** [comment]
> I ran into this issue recently. Tried to sign out and sign back in, but that didn't work. I'm starting to look at other solutions though as Postman has become less reliable.

Yeah, i use https://insomnia.rest, i moved a year ago and never looked back

**Comment by MC-sanderv** [comment]
Local sesstion/auth state corruption is a commonly reported issue on Postman. 
Usually sign out -> sign in solves the issue or manually clearing AppData on Postman files as a drastic measure. 

I will close this specific issue for now since its a commonly reported issue. 
Should there still be issue with this, please open a new ticket.
